ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/PreviewProjectTemplate.cshtml

```diff
@@ -102,10 +102,10 @@
 	</header>
 	<div class="app-summary-card__body">
 		<dl class="govuk-summary-list">
-            <govuk-summary-list-row name="governing-body-resolution" label="Governing Body Resolution" asp-route-id="@Model.Project.Id" value="@Model.Project.GoverningBodyResolution" asp-page="@Links.LegalRequirements.GoverningBodyResolution.Page" hidden-text="governing-body-resolution" />
-            <govuk-summary-list-row name="consultation" label="Consultation" asp-route-id="@Model.Project.Id" value="@Model.Project.Consultation" asp-page="@Links.LegalRequirements.Consultation.Page" hidden-text="consultation" />
-            <govuk-summary-list-row name="diocesan-consent" label="Diocesan Consent" asp-route-id="@Model.Project.Id" value="@Model.Project.DiocesanConsent" asp-page="@Links.LegalRequirements.DiocesanConsent.Page" hidden-text="diocesan-consent" />
-			<govuk-summary-list-row name="foundation-consent" label="Foundation Consent" asp-route-id="@Model.Project.Id" value="@Model.Project.FoundationConsent" asp-page="@Links.LegalRequirements.FoundationConsent.Page" hidden-text="foundation-consent" />
+            <govuk-summary-list-row name="governing-body-resolution" label="Governing Body Resolution" asp-route-id="@Model.Project.Id" value="@Model.Project.GoverningBodyResolution.SplitPascalCase()" asp-page="@Links.LegalRequirements.GoverningBodyResolution.Page" hidden-text="governing-body-resolution" />
```

Comment:
You should be able to use - https://github.com/DFE-Digital/a2b-internal/blob/3090ea47080b97d6be6fb9a7b9b0eed141d1c485/ApplyToBecomeInternal/ApplyToBecomeInternal/Extensions/EnumExtensions.cs#L8